### ISSUE TITLE
Compile error in proxy.ts - error TS2304: Cannot find name 'Clipboard'.`

### ISSUE THREAD

**Issue by mikes-gh** [DonJayamanne/gitHistoryVSCode#60]
- VSCode Version:1.7.1
- OS Version:Windows 7 sp1

Steps to Reproduce:

1.  Clone repo
2.  npm install (npm version 3.10.8)
3.  F5 to compile (tsc 1.0.3.0)

gives
`src/browser/proxy.ts(33,25): error TS2304: Cannot find name 'Clipboard'.`

It all runs as normal so I guess its just a typing issue.

looking at the npm install tree it seems the type definitions for clipboard are in 
`.\node_modules\@types\clipboard\index.d.ts`

If I add

`import * as Clipboard from 'clipboard';`
 
at the top of proxy.ts

compile succeeds.

I am a newbie to typescript so I may be missing something.

If this is indeed a fix I can create a PR.
Let me know.

**Comment by DonJayamanne** [comment]
not getting this issue. Hmm.

Try adding the following line instead (to the top):
```javascript
/// <reference path="....../node_modules/@types/clipboard/index.d.ts" />
```
Ensure the relative path is valid.

**Comment by mikes-gh** [comment]
I tried
`/// <reference path="../../node_modules/@types/clipboard/index.d.ts" />`

Path is resolved OK
Clipboard is underlined in red as unresolved.
Same as before the change.

What is your output of tsc -v

**Comment by mikes-gh** [comment]
Never mind I see it would use tsc from npm typescript package.
So to clarify from the repo root if you do

`npm run compile`

you don't get any errors?

**Comment by mikes-gh** [comment]
I've looked very where and cant see how Clipboard can be used without importing it.
Do you have the opportunity to clone to a different machine and compile?

**Comment by mikes-gh** [comment]
Ok I think I found the problem.

Clipboard typing package was updated  3 days ago

The commit below changes clipboard to cjs module.

https://github.com/DefinitelyTyped/DefinitelyTyped/commit/c5ea2b4c69f6f6a8f8346645630c805773f628b1

Since the version went from 1.5.28 to 1.5.30.

AND

"@types/clipboard": "^1.5.28" allows this update.

Things broke.

Really the package should have been major version as it is breaking change.

So how to deal with this?

"@types/clipboard": "1.5.28" 

note no ^

or 

add the imports as suggested in my first post.

I'll do the PR if you like once we decide.
